### ISSUE TITLE
like button form post

### ISSUE THREAD

**Issue by lodewiges** [csvalpha/amber-api#425]
https://csvalpha.nl/forum/categories/3/threads/298/posts

feature request,

adding a like button to form posts, and adding this as a sort option

**Comment by DrumsnChocolate** [comment]
yeah I've had this idea for a while too. We can do this more generally, for photo comments too. And for pictures. But that's a big feature
